bundle/manifests/cluster-logging.v5.0.0.clusterserviceversion.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.5.0-0 <4.7.0"
+    olm.skipRange: ">=4.5.0-0 <5.0.0"
```

Comment:
@ewolinetz @sichvoge do we want to force users through 4.6 for cases like deprecations: removing curator, removing fluent init container, etc?

Reply:
i think we should, yes. lets start to limit the upgrade path via skipRange

Reply:
@vimalk78 ^

Reply:
updated